spec/features/create_draft_spec.rb

```diff
@@ -31,6 +31,14 @@
       expect(page).to have_content('edited')
       click_on 'Revert Draft'
       expect(page).to have_content('published')
+      # Test to see if the draft has been deleted after saving the work
+      click_on 'Save Draft'
```

Comment:
I'm not sure I'm following the order of the test setup. After `Revert`, it seems like we want to `fill_in` *before* saving another draft. Am I missing something?

Reply:
This feature test isn't testing the actual values of the draft, so you don't necessarily need to have changes on that second 'Save Draft'. I added the `fill_in` 's below this because Capybara was going through the process too quickly to pass.

Reply:
ha! Okay. That feels slightly hacky, but I'm happy to let it pass for now.